SR2020/2020-03-16-competition-postpone.md

```diff
@@ -0,0 +1,35 @@
+---
+to: sr2020-teams
+subject: Postponing Student Robotics 2020
+---
+
+Hello all,
+
+Following guidance from the UK government and the University of Reading, we will need to postpone Student Robotics 2020 from the original date in April, until some time late June / early July.
+
+## Competition
+
+We are currently investigating the following dates for the competition:
+
+- 27th - 28th June
+- 4th - 5th July
+- 11th - 12th July
+
+Please fill in the following poll to let us know your preferred dates: https://forms.gle/21x6F5EBMAtu6GKh9
+
+The event will still take place over a weekend, at the University of Reading.
+
+We hope to confirm the updated date in the next week, so please let us know as soon as you can.
+
+## Tech Days
+
+There are no current planned tech days. Once things have settled down, we can organise some.
```

Comment:
Is this realistic given the current government advice?

If so, perhaps use "may" or "hope to" rather than "can".